services/webos-service-remote/service.js

```diff
@@ -0,0 +1,14 @@
+import { Handle } from './sshbus';
+import { Subscription } from './subscription';
+
+export class Service {
```

Comment:
I think some short description of what this submodule is supposed to do would make sense. (either here or in `index.js`)

Reply:
Added!